renojvarghese/recipe-app-cll
# Adding Router and App components

Added a router to go between the following different pages in the application:
http://localhost:3000/
http://localhost:3000/create-account
http://localhost:3000/dashboard
http://localhost:3000/dashboard/settings

To test:
1) `git checkout master` not necessary, but I personally like to always head back to master branch
2) `git pull` for the latest changes
3) `git checkout feature/react-router`
4) There should a folder called "react-app-cll" Make sure you are in the folder with a file called package.json. That folder is where the react app is.
5) `npm install` to get all the dependencies
6) `npm start` to start local dev environment
7) Go to the different URLs and verify that it works
 
Screenshots of what you might see:
<img width="1279" alt="screen shot 2018-12-27 at 5 44 40 pm" src="https://user-images.githubusercontent.com/24216993/50496619-2d21a180-09ff-11e9-9f83-9c3df98927b5.png">
<img width="1280" alt="screen shot 2018-12-27 at 5 45 29 pm" src="https://user-images.githubusercontent.com/24216993/50496638-53474180-09ff-11e9-890c-02753da994fe.png">
<img width="1278" alt="screen shot 2018-12-27 at 5 44 24 pm" src="https://user-images.githubusercontent.com/24216993/50496620-301c9200-09ff-11e9-887b-95d6cfa50a4d.png">
<img width="1273" alt="screen shot 2018-12-27 at 5 45 44 pm" src="https://user-images.githubusercontent.com/24216993/50496636-517d7e00-09ff-11e9-896b-485a07802503.png">
